src/cascadia/TerminalApp/TerminalWindow.cpp

```diff
@@ -1114,13 +1100,16 @@ namespace winrt::TerminalApp::implementation
             auto& parsedArgs = _appArgs->ParsedArgs();
             auto& actions = parsedArgs.GetStartupActions();
 
-            _root->ProcessStartupActions(actions, false, _appArgs->CurrentDirectory(), _appArgs->CurrentEnvironment());
-
-            if (parsedArgs.IsHandoffListener())
+            if (auto conn = args.Connection())
             {
-                _root->SetInboundListener(true);
+                _root->CreateTabFromConnection(std::move(conn));

```

Comment:
when will we get a connection in an Args versus in `SetStartupCommandline` where it will get moved into `_startupConnection`?

Is this a case where handoff (from cold start) is differnet from handoff (already running)?